index.html

```diff
@@ -1107,7 +1101,7 @@ <h2>Indexing</h2>
 is absent or exactly 8 bits deep or every pixel is either fully
 transparent or fully opaque, then an alternative representation
 called indexed-colour may be more efficient for encoding.
-Each pixel is replaced by an index into a palette.
+In the <dfn id="3indexing">indexing</dfn> transformation, each pixel is replaced by an index into a palette.
```

Comment:
Can we keep this comment here:
`<!-- Maintain a fragment named "3indexing" to preserve incoming links to it -->`

Reply:
Addressed at 0bfd41310a2a69b13be8af8b3ec8f5ea4ae1899c